packages/webview_flutter_lwe/tizen/src/webview.cc

```diff
@@ -20,6 +20,9 @@
 #include "webview_factory.h"
 
 static constexpr size_t kBufferPoolSize = 5;
```

Comment:
Would you enclose the lines 22-152 with an anonymous namespace and remove `static`? (see webview_flutter_tizen)

Reply:
done